modules/solid_mechanics/test/tests/j_integral/tests

```diff
@@ -1,92 +1,85 @@
 [Tests]
  [./j_2d]
-   type = 'Exodiff'
+   type = 'CSVDiff'
    input = 'j_integral_2d.i'
-   exodiff = 'j_integral_2d_out.e'
-   custom_cmp = 'j_integral_2d.cmp'
+   csvdiff = 'j_integral_2d_out.csv'
  [../]
  [./j_2d_points]
-   type = 'Exodiff'
+   type = 'CSVDiff'
    input = 'j_integral_2d_points.i'
-   exodiff = 'j_integral_2d_points_out.e'
+   csvdiff = 'j_integral_2d_points_out.csv'
  [../]
  [./j_2d_mouth_dir]
-   type = 'Exodiff'
+   type = 'CSVDiff'
    input = 'j_integral_2d_mouth_dir.i'
-   exodiff = 'j_integral_2d_mouth_dir_out.e'
+   csvdiff = 'j_integral_2d_mouth_dir_out.csv'
  [../]
  [./j_2d_topo_q]
-   type = 'Exodiff'
+   type = 'CSVDiff'
    input = 'j_integral_2d_topo_q_func.i'
-   exodiff = 'j_integral_2d_topo_q_func_out.e'
```

Comment:
I'd actually like to keep the exodiff to check the q functions on some of these tests. Maybe we should do both exodiff and csvdiff, and just check the q functions with exodiff so that they're not sensitive to small stress differences.